src/DocumentationBrowserViewExtension/ScriptingObject.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Dynamo.DocumentationBrowser
+{
+
+    [ClassInterface(ClassInterfaceType.AutoDual)]
+    [ComVisible(true)]
+    public class ScriptingObject
+    {
+        private DocumentationBrowserViewModel viewModel;
+
+        public ScriptingObject(DocumentationBrowserViewModel viewModel)
+        {
+            this.viewModel = viewModel;
+        }
+
+        /// <summary>
+        /// The method takes a string code and will execute a different program based on that code
+        /// </summary>
+        /// <param name="message"></param>
+        public void Notify(string message)
+        {
+            if (string.Equals(message, "insert"))
+            {
+                // Insert the graph inside the current worskspace
+                this.viewModel.InsertGraph();
```

Comment:
@dnenov do you remember if this functionality of inserting the graph from the DocumentationBrowser was tested also with xml dyn files (most of the dyn files in fallback_docs folder are in json format but there are some cases in which we are using xml files but is crashing when doesn't have notes).
FYI: @QilongTang 
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/0de33a36-b0ff-42fc-9ecf-66a15ea622bb)

Reply:
You are right, @RobertGlobant20 , I don't think there was anything to test with regarding the XML files. I do remember trying to find valid XML dynamo files to test with and failing (Dynamo itself wouldn't open them).